site/source/docs/api_reference/stack.h.rst

```diff
@@ -0,0 +1,39 @@
+.. _stack-h:
+
+=======
+stack.h
+=======
+
+The functions defined in `<emscripten/stack.h>` allow inspecting
+information about the WebAssembly data stack. This data stack is
+the data contained within the linear memory.
+
+.. c:function:: uintptr_t emscripten_stack_get_base(void)
+
+  Returns the starting address of the wasm stack. This is the address
```

Comment:
Why use `wasm stack` here but `data stack` above?   I think maybe just `stack` would be enough here given the above.

Also we have sometimes called this stack the `shadow stack` although some folks have objected to that use of the term shadow.

Reply:
I'd copied this text from the header, so I've updated it in both places. Thanks!